android/src/main/java/com/neuroidreactnativesdk/NeuroidReactnativeSdkModule.kt

```diff
@@ -2,13 +2,13 @@ package com.neuroidreactnativesdk
 
 import android.app.Application
 import com.facebook.react.bridge.*
+import com.facebook.react.bridge.Arguments
```

Comment:
import com.facebook.react.bridge.*
import com.facebook.react.bridge.Arguments

Should we remove the wildcard import on bridge.* to reduce package size? If not, we can remove the `Arguments` since this is redundant.